integrations/adobe-analytics/lib/index.js

```diff
@@ -216,14 +298,117 @@ AdobeAnalytics.prototype.initialize = function() {
     });
   }
 };
+function chromecastHeartbeatVideoStart(track) {
```

Comment:
Let's put all of the heartbeat functions in a separate file to try to keep this lines of code in this file more manageable.

e.g. a file named `heartbeat.js` can have all of these functions `export`ed and then have this `lib/index.js` import and call them appropriately.

Reply:
@mericsson is this just for the chromecast hb events or also including the original hb events?

Reply:
I've moved out the chromecast hb events to a new file, I feel a little uncomfortable making such a dramatic change to the existing hb events, but let me know if that's what is best here.